src/main/cli/cli.c

```diff
@@ -4242,6 +4242,9 @@ static void cliSave(const char *cmdName, char *cmdline)
     UNUSED(cmdline);
 
     if (tryPrepareSave(cmdName)) {
+#ifdef USE_SIMPLIFIED_TUNING
+        applySimplifiedTuning(currentPidProfile);
```

Comment:
This change will result in the simplified tuning setting being used to overwrite any tuning settings on every save, even if the simplified tuning settings have not been modified.
This means that it will be completely impossible for users to tune their craft with anything but simplified settings, or backup / restore any such configuration.
Instead of overwriting the tuning settings with values calculated from the simplified settings _on every `save`_ in CLI, users should use `apply_simplified_tuning` before saving to explicitly overwrite their tuning settings with simplified settings.

Reply:
Yes, this is probably not required now that the Configurator side has changed so that it reverts the numbers if changes to the sliders are not saved.
It is not as harmful as you suggest.
`applySimplifiedTuning` only changes values for which the sliders would be dominant.  If the slider mode is off, or for yaw when slider mode is RP, the non-slider-controlled PID or filter numbers are not changed by slider positions, whether those changes are typed into CLI of hanging over from some previous slider position.  The user can change values in Configurator directly, via OSD or via CLI, and the changes will stick.  
What this line achieves is certainty that the slider position and configuration from a snippet pasted into the CLI or applied via the preset system will always generate the intended PID values.  
This is really important.  
I have tested the above and it worked correctly.
If not then we may need to modify `applySimplifiedTuning` - certainly that code should only change values for which the sliders 'have control'.  And in that case, we want it to change values that sliders control if those slider positions are edited in the CLI.
At least that's my take on it.